compiler/luci/service/src/Nodes/CirclePad.test.cpp

```diff
@@ -90,3 +90,61 @@ TEST(ShapeRuleTest, pad_without_padding_NEG)
 
   ASSERT_ANY_THROW(shape_inf_rule.infer(node_pad, shape));
 }
+
+TEST(ShapeRuleTest, pad_non_const_paddings)
```

Comment:
@zetwhite 
I am wondering if additional test cases are needed.

Reply:
> I am wondering if additional test cases are needed.

I think current test is enough :)

Reply:
> 
> I think current test is enough :)

Thank you for the review!
 
I'll make a PR with this TC, then.

Reply:
:+1: